src/connectors/voice-search/connectVoiceSearch.ts

```diff
@@ -77,6 +90,20 @@ const connectVoiceSearch: VoiceSearchConnector = (
           const setQueryAndSearch = (query: string) => {
             if (query !== helper.state.query) {
               previousQuery = helper.state.query;
+              if (typeof additionalQueryParameters === 'function') {
+                const queryLanguages = language
+                  ? [language.split('-')[0]]
+                  : undefined;
+                helper.setState(
+                  helper.state.setQueryParameters({
+                    queryLanguages,
```

Comment:
Is there any reason why `queryLanguages` is set only when there's `additionalQueryParameters`?

Reply:
It might counteract with queryLanguages set with a `configure`, so it should be something you enable. Also I found it easier to implement that way